### ISSUE TITLE
Accessibility: Dashboard cannot be used with Emacspeak

### ISSUE THREAD

**Issue by devinprater** [emacs-dashboard/emacs-dashboard#295]
Hello. I am a blind person, and use [Emacspeak](https://www.github.com/tvraman/emacspeak) to use Emacs (on arch Linux, Emacs version GNU Emacs 27.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.22, cairo version 1.17.3) of 2020-08-28)

I installed Dashboard with Melpa, and used:

``` elisp
( require   'dashboard )
(dashboard-setup-startup-hook)
```

to initialize it on startup. But, using Emacspeak, pressing C-n and C-p don't work. Looking further, it turns out Dashboard-mode has its own previous-line and next-line commands. I'm not a programmer, so I can't really say how to fix this, but what Emacspeak usually does is use Advise to speak info, like in [Emacspeak-dired](https://www.github.com/tvraman/emacspeak/blob//lisp/emacspeak-dired.el). I hope this helps, and thanks so much for the package!

**Comment by jcs090218** [comment]
Hi @devinprater. You can use arrow keys to navigate through the dashboard buffer. We do have the proposal at #187. Let me know if the need anything else! :)

**Comment by devinprater** [comment]
The problem is that neither "dashboard-next-line" nor "dashboard-previous-line" work with Emacspeak. So none of the keys bound to those commands result in the items moved to being spoken by Emacspeak.

**Comment by jcs090218** [comment]
Not sure if I understand your question. Would `next-line` and `previous-line` command work in Emacspeak? If so, try override the key binding like the following snippets.

```el
(define-key dashboard-mode-map (kbd "C-p")  #'previous-line)
(define-key dashboard-mode-map (kbd "C-n")  #'next-line)
```

**Comment by jcs090218** [comment]
I think the issue can be closed? Feel free to reopen it if you still having issue. :)
